packages/nimble-components/src/rich-text-mention/base/index.ts

```diff
@@ -45,6 +45,9 @@ export abstract class RichTextMention<
     @attr
     public pattern?: string;
 
+    @observable
+    public buttonLabel?: string;
+
```

Comment:
This is a user facing attribute. The idea is that when configuring the mention youu set this property


```suggestion
    @attr({ attribute: 'button-label' })
    public buttonLabel?: string;

```

This will need to be set in the anguular directive as well

Reply:
Updated as per your suggestion in components and created a PR for Angular directive: https://github.com/ni/nimble/pull/1706